_includes/tutorials/kafka-connect-datagen-local/kafka/code/Dockerfile

```diff
@@ -0,0 +1,5 @@
+FROM confluentinc/cp-kafka-connect-base:5.5.1
+
+ENV CONNECT_PLUGIN_PATH="/usr/share/java,/usr/share/confluent-hub-components"
+
+RUN confluent-hub install --no-prompt confluentinc/kafka-connect-datagen:latest
```

Comment:
FYC Use an explicit version to prevent any unexpected compatibility issues down the line for users

Reply:
I'll pin this to `0.3.2`